virt/virt-2-4-release-notes.adoc

```diff
@@ -8,56 +8,105 @@ toc::[]
 
 include::modules/virt-what-you-can-do-with-virt.adoc[leveloffset=+2]
 
-=== {VirtProductName} support
+[id="virt-2-4-new"]
+== New and changed features
+. Placeholder for new content.
```

Comment:
Not a big deal but this would be better as a comment (preceded by `//` )

Reply:
Changed the text to a comment with //.

Reply:
Changed to a comment preceded by //